.vscode/settings.json

```diff
@@ -1,23 +1,22 @@
 {
-    "[c]": {
-        "editor.formatOnSave": true
-    },
-    "terminal.integrated.env.windows": {
-        "PATH": "${env:PATH}"
-    },
-    "editor.tokenColorCustomizations": {
-        "[Default Dark+]": {
-            "textMateRules": [
-                {
-                    "scope": "constant.other.placeholder",
-                    "settings": {
-                        "foreground": "#CE9178"
-                    }
-                }
-            ]
-        }
-    },
-    "terminal.integrated.tabs.enableAnimation": false,
-    "files.autoGuessEncoding": true,
-    }
+  "[c]": {
```

Comment:
This one's always a fight. My vscode keeps trying to overwrite it 🫠

Reply:
We have the same issue on our end too, and don't have a fix just yet. Could you squash this particular commit?